src/lib.rs

```diff
@@ -0,0 +1,198 @@
+//! # Substrate
+//!
+//! Substrate is a Rust framework for building blockchains in a modular and extensible way. While in
+//! itself un-opinionated, it is the main engine behind the Polkadot ecosystem.
+//!
+//! [![github]](https://github.com/paritytech/substrate/) - [![polkadot]](https://polkadot.network)
+//!
+//! This crate in itself does not contain any code and is just meant ot be a documentation hub for
+//! substrate-based crates.
+//!
+//! ## Overview
+//!
+//! Substrate approaches blockchain development with a focus on **safety**, **correctness**, and
+//! **upgradeability**.
+//!
+//! **Safety** is acquired through a use of Rust, a modern language empowering everyone to build
+//! reliable and efficient software.
```

Comment:
Can you maybe explain a bit closer what is meant with `Safety` in the first place?  
I assume it alludes to panic-free and deterministic code that is easily auditable.

Reply:
I mostly want to lean into the type of memory safety guarantees that Rust brings us here.